src/main/java/org/commcare/resources/model/ResourceTable.java

```diff
@@ -872,14 +872,13 @@ public void clearUpgrade(CommCarePlatform platform) {
     // Clears all resources in the table
     public void clearAll(CommCarePlatform platform) {
         clearByStatus(platform, RESOURCE_STATUS_ALL_RESOURCES);
-        storage.removeAll();
```

Comment:
Moving this to `clearByStatus` since both `clearAll` and `clearUpgrade` calls `clearByStatus` and we need `storage.removeAll` in code paths for both `clearAll` and `clearUpgrade`.